### DISCUSSION TITLE
How to uninstall after installing from source

### DISCUSSION THREAD

**Discussion by OlaoluwaM** [Aylur/ags#149] [Q&A] [answered]
Looking to switch from the Copr to building from source, but would like to know how (if possible) I can uninstall after building and installing from source. Thanks!

**Comment by Aylur** [comment, marked as answer]
you can of course, meson will install to /usr/local/bin and /usr/local/share, when running `meson install` it will print to the console where it installed, you can just `sudo rm` the installed files after
